modules/openy_traction_rec_membership/src/MembershipImporter.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
do we really need strict mode in a such simple file?

Reply:
@anpolimus For now maybe not, but it just was done according to code sniffer recommendations and could be useful in future